### PR TITLE
Use umb-icon in umb-node-preview

### PR DESCRIPTION
**Pull request by bjarnef**
### Prerequisites

- [x] I have added steps to test this contribution in the description below

### Description
We can use the `<umb-icon>` inside the `<umb-node-preview>` component, so we should be able to use custom SVG icons in this component as well.

### CODE REVIEW

**Comment by poornimanayar** [comment]
Hey @bjarnef ,

How is this different from https://github.com/umbraco/Umbraco-CMS/pull/9013 ?

Poornima

**Comment by poornimanayar** [comment]
Haha exactly why I asked you 😆 What can we do to achieve best of both worlds?

Poornima

**Comment by bjarnef** [comment]
@poornimanayar could you have a look at your current PR and then with this change applied?
https://github.com/umbraco/Umbraco-CMS/commit/69f883625c87c3706b1b5136702edb2076a1e92a

You should be able to see a difference in icon size. Then I can close this PR.

**Comment by poornimanayar** [comment]
Hi @bjarnef ,

Yes, I can apply the change to my PR and test 👍 

Poornima

**Comment by nathanwoulfe** [comment]
Hey @poornimanayar / @bjarnef where are we at with this one? Is this PR still required, or does #9013 replace it?

**Comment by bjarnef** [comment]
@nathanwoulfe I think the PR from @poornimanayar should be fine in https://github.com/umbraco/Umbraco-CMS/pull/9013

At the time I submitted this PR I also removed this from `umb-node-preview__icon` class:
https://github.com/umbraco/Umbraco-CMS/commit/69f883625c87c3706b1b5136702edb2076a1e92a

```
width: 25px;
min-height: 25px;
```

since the `<umb-icon>` itself handle the icon sizing and with the css above, the icons in `umb-node-preview` was smaller compared to other icons.

**Comment by nathanwoulfe** [comment]
@bjarnef looks like I'd already merged @poornimanayar's PR, but will merge this too, to catch the CSS change. Given #9013 is merged, there's now a conflict in this PR - I'd resolve that for you, but git is giving me grief when trying to push to PR branches.

File: src/Umbraco.Web.UI.Client/src/views/components/umb-node-preview.html [hunk 1]
```diff
@@ -1,85 +1,75 @@
 <div class="umb-node-preview" ng-class="{'umb-node-preview--sortable': sortable, 'umb-node-preview--unpublished': published === false }">
     <div class="flex"> <!-- div keeps icon and nodename from wrapping -->
-        <i ng-if="icon" class="umb-node-preview__icon {{ icon }}" aria-hidden="true"></i>
+        <umb-icon ng-if="icon" icon="{{icon}}" class="umb-node-preview__icon {{icon}}"></umb-icon>
```
Comment (nathanwoulfe) [inline src/Umbraco.Web.UI.Client/src/views/components/umb-node-preview.html:3]: The bound `{{icon}}` property in the class attribute can be removed too, it's not required.

**Review by nathanwoulfe — CHANGES_REQUESTED** [review]

**Review by nathanwoulfe — APPROVED** [review]

**Pull request merged**